apps/src/applab/dontMarshalApi.js

```diff
@@ -277,6 +318,8 @@ const interpreterFunctions = {
   insertItem: interpreterInsertItem,
   removeItem: interpreterRemoveItem,
   appendItem: interpreterAppendItem,
+  getValue: interpreterGetValue,
```

Comment:
Sorry for the late question, but just curious: is `getValue` added to the global namespace for code in applab?  If so, could it be too generic, and take away another potential use of that name?

Reply:
We don't have any other pre-defined uses of `getValue` besides this block, but I see what you mean in that "value" is specific to "key: value pairs" but is also used generically to talk about all kinds of things. Hmm...

 I'm not 100% sure this answers your question, but I was trying to figure it out and it looks like you can write a function with the same name as any of the blocks and it will overwrite the block function. App Lab will take the user written function definition over the block. I don't know if that's good or bad, but I'm convinced `getValue` is at least consistent in terms of how it behaves. 

![Screen Shot 2021-04-14 at 2 25 15 PM](https://user-images.githubusercontent.com/12300669/114761818-ffc3a380-9d2e-11eb-890e-e61cd9da15b0.png)

Reply:
Yeah, I wonder if something like `getObjectValue` might be a bit more appropriate, but I defer to others for expertise on this.

Reply:
To Brendan's original question- yes, it's added to a global namespace in the interpreter. So we would not be able to make another block called `getValue()` with different behavior later.
Renaming blocks later is hard because we don't have a way to update all existing student projects that might already use the old name.

@Erin007 would `getObjectValue` work for your use cases? It's also a little more similar to the naming style of `getKeyValue`